modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/portlet/SearchPortlet.java

```diff
@@ -66,6 +66,10 @@
 )
 public class SearchPortlet extends MVCPortlet {
 
+	/**
+	 * @deprecated As of Judson (7.1.x), replaced by search pages and widgets
```

Comment:
Should it be 7.4?

Reply:
Hm, the ticket mentions it happens in 7.1 https://help.liferay.com/hc/en-us/articles/360017897432-Deprecated-Apps-in-7-1-What-to-Do#foundation?

Reply:
@gustavolimav shall mark this resolved! 😅 let me know if that's not ok, but thanks for checking as well